my-website/docs/competicoes/sacup/rulesnrequirements.md

```diff
@@ -6,6 +6,172 @@ sidebar_label: Rules & Requirements
+- As atualizações de progresso devem conter o progresso das características técnicas do material durante o desenvolvimento. 
+- Apenas as equipes cujas aplicações cumprem as medidas serão avaliadas para entrar na competição. 
+- As equipes aceitas serão anunciadas pelo Master Schedule e receberão uma ID de equipe.
+
+# Relatório Técnico do Projeto
+
+- Cada equipe deverá submeter um relatório técnico que contenha uma visão global do projeto para o painel de jurados e outros oficiais da competição.
+-  O Relatório deve ser formatado de acordo com o guia da American Institute of Aeronautics and Astronautics (AIAA), usando o template do Microsoft® Word.
+- O Template está disponível [neste link](http://www.soundingrocket.org/sa-cup-documents--forms.html). 
+- A equipe deve submeter antes ou na data especificada apenas um arquivo PDF do Relatório Técnico do Projeto.
+- Para Relatórios Técnicos que excedam 50 Mb deve-se fazer  o upload em um servidor na nuvem, desde que as permissões permitam que os jurados tenham acesso irrestrito ao documento. 
+- [Link para submissão do Relatório](https://www.herox.com/SpaceportAmericaCup2021).
+- As equipes devem levar uma certa quantidade de cópias para o Spaceport America Cup para que os jurados e outros oficiais da competição possam consultar a qualquer momento durante as interações com a equipe. 
+- O Relatório deve seguir o seguinte modelo de título: “Team Project Technical Report to the Spaceport America Cup".
+Abstract 
+- O Relatório Técnico do Projeto deve conter um Abstract. Este deve identificar a categoria/ missão do veículo de lançamento, em que equipe está competindo, identificar quaisquer características diferenciais do veículo de lançamento, definir a missão do payload e provem quaisquer informação adicional necessária.
+
+## Introdução 
+
+O Relatório Técnico do Projeto deve conter uma Introdução. Essa seção deve conter um overview do programa acadêmico, stakeholders, estrutura da equipe e estratégia de gestão da equipe.
+
+## Overview da Arquitetura do Sistema
+- O Relatório Técnico do Projeto deve conter um Overview da Arquitetura do Sistema. 
+- Esta seção deve começar com um overview do sistema integrado, incluindo uma figura de corte do veículo de lançamento e seus subsistemas principais. Esta descrição deverá ser seguida das seguintes sub-seções:
+
+   - Subsistema da Propulsão
+   - Subsistema de aero estruturas
+   - Subsistema de Recuperação 
+   - Subsistema de Payload 
+
+- Cada subseção deverá conter uma descrição detalhada de cada subsistema e reflexão sobre as decisões técnicas, de design e de manufatura.
+- Desenhos técnicos dos subsistemas devem ser inclusos no apêndice especificado.  
+
+## Conceito da Missão e Overview de Operações
+
+- O Relatório Técnico do Projeto deve conter o Overview de Conceito de Operação da Missão (CONOPS).
```

Comment:
Esses nomes de relatórios são bem mais comuns de serem vistos em inglês. Pode traduzir, mas faz assim: 
"O Project Technical Report (Relatório Técnico do Projeto, também chamado de PTR) deve conter o Mission Concept of Operations Overview (Overview de Conceito de Operação da Missão, também chamado de CONOPS)."